base/java-tools/src/com/netscape/cmstools/CMCResponse.java

```diff
@@ -352,8 +353,7 @@ public static void main(String args[]) throws Exception {
 
         CommandLine cmd = parser.parse(options, args, true);
 
-        @SuppressWarnings("unused")
-        String database = cmd.getOptionValue("d");
+        String dbdir = cmd.getOptionValue("d");
```

Comment:
here you changed the variable name from "database" to "dbdir".  I  I don't see any further change of the name from "database" to "dbdir".  Would this not break anything?

Reply:
The variable database wasn't used anyhere.